app/models/notifications/NotificationFrontendModel.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.notifications
+
+import java.time.Instant
+
+import play.api.libs.json.Json
+
+case class NotificationFrontendModel(
```

Comment:
Can we  just call it `Notification`?

Reply:
Notification is the model used by backend service to store notifications in the DB. To communicate with frontend it uses different model named `NotificationFrontendModel`. So to be more clear that these models are the same it is called the same on the FE.

Reply:
Not in the FE. But for clarity with naming we are using same names as the BE does.